Bottom: 27f95411bd9b97810ea6432e988e53aa0a78fa32
Top:    57c4f07f976a75c65a0149722a7b729c249084c2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 15:38:24 +0100

draw.c: Only set Gathering during hid_expose_callback()

Removes setting of Gathering to false prior to calling UpdateAll()
in ClearAndRedrawOutput(), and removes setting of Gathering to
true after calling gui->invalidate_all() in Redraw()


---

diff --git a/src/draw.c b/src/draw.c
index c82f25a..1415b2e 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -211,7 +211,6 @@ RedrawOutput (BoxTypePtr area)
 void
 ClearAndRedrawOutput (void)
 {
-  Gathering = false;
   UpdateAll ();
 }
 
@@ -227,7 +226,6 @@ static void
 Redraw (bool ClearWindow, BoxTypePtr screen_area)
 {
   gui->invalidate_all ();
-  Gathering = true;
 }
 
 static int
